src/modules/ekf2/EKF2.hpp

```diff
@@ -124,14 +124,21 @@ class EKF2 final : public ModuleParams, public px4::ScheduledWorkItem
 	static bool trylock_module() { return (pthread_mutex_trylock(&ekf2_module_mutex) == 0); }
 	static void unlock_module() { pthread_mutex_unlock(&ekf2_module_mutex); }
 
-	bool multi_init(int imu, int mag);
+	bool multi_init(int imu, int mag, uint8_t vis_mode);
 
 	int instance() const { return _instance; }
 
 private:
 
 	static constexpr uint8_t MAX_NUM_IMUS = 4;
 	static constexpr uint8_t MAX_NUM_MAGS = 4;
+	static constexpr uint8_t MAX_NUM_VIS_GNSS = 2;
```

Comment:
Perhaps a "must be 2" comment here, other values does not make sense?

Reply:
Other values could make sense if we want to compare more possible combinations, e.g. GNSS/Vision/Both. But that would also require changes to the multi EKF spawning logic. I added a comment for now.